README.md

```diff
@@ -2,22 +2,6 @@
 
 A new and improved cluster mass modeling code descended from [clmassmod](https://github.com/LSSTDESC/clmassmod)
 
-## Motivation
```

Comment:
I think you did a bad merge here at some point. Your PR is trying to reset the README.